packages/autorest.python/autorest/codegen/templates/model_base.py.jinja2

```diff
@@ -570,6 +570,36 @@ class Model(_MyMutableMapping):
             result[k] = Model._as_dict_value(v, exclude_readonly=exclude_readonly)
         return result
 
+{% if code_model.has_form_data %}
+    def as_origin_dict(self, *, exclude_readonly: bool = False) -> typing.Dict[str, typing.Any]:
+        """Return a dict that the value is the origin value instead of serialized value.
```

Comment:
Values of `as_dict` are serialized values however we want original value of body so I add `as_origin_dict`

Reply:
this is public model property that we shouldn't have. Why do we need this publicly exposed to users on a model?

Reply:
Make sense and we shall hide it.